tests/TestCase.php

```diff
@@ -13,12 +13,15 @@
 use Tipoff\Locations\LocationsServiceProvider;
 use Tipoff\Statuses\StatusesServiceProvider;
 use Tipoff\Support\SupportServiceProvider;
+
 use Tipoff\TestSupport\BaseTestCase;
 use Tipoff\TestSupport\Providers\NovaPackageServiceProvider;
 
+
 class TestCase extends BaseTestCase
 {
     protected bool $stubNovaResources = false;
```

Comment:
@huntermontell Every package should have a testcase that extends the BaseTestCase and has $stubNovaResources defined.